queries/cmake/highlights.scm

```diff
@@ -0,0 +1,87 @@
+(bracket_content) @parameter
+(quoted_element) @parameter
+(unquoted_argument) @parameter
+
+(variable) @variable
+
+[
+  (normal_command)
+  (if_command)
+  (elseif_command)
+  (else_command)
+  (endif_command)
+  (foreach_command)
+  (endforeach_command)
+  (while_command)
+  (endwhile_command)
+  (function_command)
+  (endfunction_command)
+  (macro_command)
+  (endmacro_command)
+  (message_command)
+]@function
+
+[
+  "1"
+  "ON"
+  "YES"
+  "TRUE"
+  "Y"
+  "0"
+  "OFF"
+  "NO"
+  "FALSE"
+  "N"
+  "IGNORE"
+  "NOTFOUND"
+  "NOT"
+  "AND"
+  "OR"
+  "COMMAND"
+  "POLICY"
+  "TARGET"
+  "TEST"
+  "DEFINED"
+  "CACHE"
+  "ENV"
+  "IN_LIST"
+  "EXISTS"
+  "IS_NEWER_THAN"
+  "IS_DIRECTORY"
+  "IS_SYMLINK"
+  "IS_ABSOLUTE"
+  "MATCHES"
+  "LESS"
+  "GREATER"
+  "EQUAL"
+  "LESS_EQUAL"
+  "GREATER_EQUAL"
+  "STRLESS"
+  "STRGREATER"
+  "STREQUAL"
+  "STRLESS_EQUAL"
+  "STRGREATER_EQUAL"
+  "VERSION_LESS"
+  "VERSION_GREATER"
+  "VERSION_EQUAL"
+  "VERSION_LESS_EQUAL"
+  "VERSION_GREATER_EQUAL" 
+  "IN"
+  "RANGE"
+  "ZIP_LISTS"
+  "LISTS"
+  "ITEMS"
+  "FATAL_ERROR"
+  "SEND_ERROR"
+  "WARNING"
+  "AUTHOR_WARNING"
+  "DEPRECATION"
+  "NOTICE"
+  "STATUS"
+  "VERBOSE"
+  "DEBUG"
+  "TRACE"
+  "CHECK_START"
+  "CHECK_PASS"
+  "CHECK_FAIL"
+] @keyword
```

Comment:
Are all of those keywords or built-in functions?

Reply:
They are kind of contextual keywords, as CMake doesn't have a way to specify named arguments, they just parse it at runtime with these keywords as markers.